### DISCUSSION TITLE
V3 - Livewire Component Autodiscovery

### DISCUSSION THREAD

**Discussion by cpbret** [filamentphp/filament#7404] [Help]
Components in App/Filament were previously auto registered by Filament in v2. 

Is this no longer a thing? Is there an easy way to do it?

<img width="805" alt="Screenshot 2023-08-02 at 9 29 46 AM" src="https://github.com/filamentphp/filament/assets/64978571/7ac83897-c5fc-4e57-bff0-b03ef2be2664">
<img width="903" alt="Screenshot 2023-08-02 at 9 29 08 AM" src="https://github.com/filamentphp/filament/assets/64978571/2bbea8d0-bcdf-4b42-ac26-a32cd3b8945e">

**Comment by leandrocfe** [comment]
`php artisan livewire:publish --config`
?

**Comment by danharrin** [comment]
Please PR a `->discoverLivewire()`, that would be a good idea!

**Reply by cpbret** [reply]
It seems like the other discovery happens at the Panel level, but I'm upgrading an app with primarily custom page components utilizing tables and forms. Unfortunately we have a feature that I believe is preventing us from adopting Panels.

I'm assuming the solution for me is probably to add back auto discovery in a service provider pointing to the previous components?

**Reply by danharrin** [reply]
If you PR `discoverLivewire()` similar to `discoverPages()` it should work with minimal effort
